Dockerfile.py2

```diff
@@ -26,7 +26,7 @@ RUN apt -y update -qq \
 
 
 ENV ANDROID_NDK_HOME="${ANDROID_HOME}/android-ndk"
-ENV ANDROID_NDK_VERSION="16b"
+ENV ANDROID_NDK_VERSION="17c"
```

Comment:
doesn't this kind of change already deserve it's own PR for instance?
That why it's easier to justify why it's needed and eventually make it early (or not) to the tree